src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
```

Comment:
The name is kinda ambiguous (most loops are "processing" of some sort). Perhaps `containsTaskForMonthAndType` or something along those lines?